### DISCUSSION TITLE
Details about Pseudoreplicates

### DISCUSSION THREAD

**Discussion by ba-davis** [GreenleafLab/ArchR#2015] [Questions / Documentation]
Can you clarify the use of pseudoreplicates in analysis with ArchR? Is the information about which cells are assigned to which pseudoreplicate available anywhere? I have checked the log files for addGroupCoverages and I have found the number of cells per psuedoreplicate, but not a full list of cell ids. I was thinking of visualizing cells by pseudoreplicate on the UMAP plot.

Also, can you clarify the meaning of the minReplicates parameter in addGroupCoverages? I am using a value of 4, but many of my groups (specified by groupBy parameter) have 2 or 3 pseudoreplicates created. From the flow chart in the vignette, it looks like my samples fall under scenario 3 and I would expect at least 4 pseudoreplicates to be created.

Thank you

**Comment by rcorces** [comment]
> Is the information about which cells are assigned to which pseudoreplicate available anywhere?

see the `returnGroups` parameter

> Also, can you clarify the meaning of the minReplicates parameter in addGroupCoverages? I am using a value of 4, but many of my groups (specified by groupBy parameter) have 2 or 3 pseudoreplicates created. From the flow chart in the vignette, it looks like my samples fall under scenario 3 and I would expect at least 4 pseudoreplicates to be created.

`minReplicates` means what is says it means but it is also dependent on other parameters like `minCells` and `sampleRatio`. If you ask for a minimum of 4 replicates from a cluster with 200 cells and you specify a minCells of 150, you arent going to get 4 replicates.
